pkg/controller/jobs/pod/pod_controller_test.go

```diff
@@ -2451,6 +2764,9 @@ func TestReconciler(t *testing.T) {
 			if diff := cmp.Diff(tc.wantWorkloads, gotWorkloads.Items, tc.workloadCmpOpts...); diff != "" {
 				t.Errorf("Workloads after reconcile (-want,+got):\n%s", diff)
 			}
+			if diff := cmp.Diff(tc.wantEvents, recorder.RecordedEvents, cmp.Transformer("sortedEvents", utiltesting.SortedEvents)); diff != "" {
```

Comment:
There is a specific opt for sorting :) https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#SortSlices

Reply:
Indeed, opened a small cleanup https://github.com/kubernetes-sigs/kueue/pull/1691